verifact/forum/schema.py

```diff
@@ -10,6 +11,13 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class AnswerNode(DjangoObjectType):
+    class Meta:
+        model = Answer
+        filter_fields = {"question_id": ["exact"]}
```

Comment:
When will this be used in practice? Wouldn't you always have the question node when you have the answer?

Reply:
true, think it's an artifact of just playing around with django. will remove from the pipfile as well